src/core/ucc_progress_queue_mt.c

```diff
@@ -35,15 +44,32 @@ static void ucc_pq_mt_enqueue(ucc_progress_queue_t *pq, ucc_coll_task_t *task)
             return;
         }
     }
+
+    ucc_pq_mt_enqueue(pq, task);
+}
+
+inline static void ucc_pq_mt_dequeue(ucc_progress_queue_t *pq,
```

Comment:
is it really inline? you pass it as fn pointer on pq->dequeue, can it be inline this way?

Reply:
I dont know. removed inline